bin/ci/ubuntu/build-and-test.sh

```diff
@@ -66,11 +66,13 @@ fi
 # if we get a crash.
 gdb -return-child-result -quiet -batch \
     -ex "set env MALLOC_CHECK_=3" \
+    -ex "set demangle-style none" \
     -ex "set print thread-events off" \
+    -ex "file $APP_PATH" \
+    -ex "set args $APP_ARGS" \
     -ex run \
     -ex "thread apply all backtrace full" \
     -ex "quit" \
-    --args $APP_PATH $APP_ARGS
```

Comment:
I like the idea of running with a debugger on Travis and getting info. Its a good approach.